src/select/dropdown.js

```diff
@@ -150,6 +154,7 @@ export default class SelectDropdown extends React.Component<DropdownPropsT> {
               Option: overrides.DropdownOption || {},
             },
           )}
+          {...statefulMenuProps}
```

Comment:
We need to omit `overrides` from the `statefulMenuProps ` since we merge it above.